hero/examples/news.ycombinator.com.ts

```diff
@@ -3,7 +3,7 @@ import agent, { Observable } from 'secret-agent';
 // process.env.SA_SHOW_BROWSER = 'true';
 
 async function run() {
-  await agent.configure({ userAgent: '~ chrome >= 88' });
+  await agent.configure({ userAgent: '~ chrome = 88.*' });
```

Comment:
Why the asterisk?

Reply:
compareVersions otherwise says nothing = 88.0.0

Reply:
Of Chrome? No... The versions are like 88.0.4324.{many patches}

Reply:
It works without the asterisk, so I removed it from the example.